README.md

```diff
@@ -86,35 +86,42 @@ yarn add effector
 Three following examples that will give you a basic understanding how the state manager works:
 
 ### Increment/decrement
+
 ```js
-import {createStore, createEvent} from "effector";
-import {createComponent} from "effector-react";
+import {createStore, createEvent} from 'effector'
+import {useStore} from 'effector-react'
 
-const increment = createEvent("increment");
-const decrement = createEvent("decrement");
-const resetCounter = createEvent("reset counter");
+const increment = createEvent('increment')
+const decrement = createEvent('decrement')
+const resetCounter = createEvent('reset counter')
 
-const counter = createStore(0)
+const $counter = createStore(0)
```

Comment:
Not sure if it is worth advising as a common practice, at least effector itself avoid this in tests and website